pkg/sfu/receiver.go

```diff
@@ -866,6 +877,7 @@ func (w *WebRTCReceiver) GetPrimaryReceiverForRed() TrackReceiver {
 		})
 		if w.primaryReceiver.CompareAndSwap(nil, pr) {
 			w.redPktWriter.Store(redPktWriteFunc(pr.ForwardRTP))
+			w.redSenderReportWriter.Store(redSenderReportWriteFunc(pr.ForwardRTCPSenderReport))
```

Comment:
are these ever not updated at the same time? could there be one value storing
```go
type forwarderThing {
    ForwardRTP(...)
    ForwardRTCPSenderReport(...)
}
```

Reply:
True, will combine them.

Reply:
There are two receivers in there (one converting from RED -> Opus, one converting from Opus -> RED). But, only one should be active. Can probably store as one. Will take a look.